pandas/io/pickle.py

```diff
@@ -140,15 +142,15 @@ def read_pickle(path, compression='infer'):
     path = _stringify_path(path)
     f, fh = _get_handle(path, 'rb', compression=compression, is_text=False)
 
-    # 1) try with cPickle
+    # 1) try with Pickle
     # 2) try with the compat pickle to handle subclass changes
```

Comment:
Minor but I suppose we don't need a three step comment here anymore

Reply:
Technically there's still a 3 step process, "compat pickle" in this case refers to compatibility with older versions of pandas and not PY2/3 compat.

Reply:
Will clarify this comment though.